app/assets/src/components/views/discovery/DiscoveryView.jsx

```diff
@@ -697,14 +698,14 @@ class DiscoveryView extends React.Component {
   };
 
   handleMapTooltipTitleClick = locationId => {
-    this.setState({ mapSelectedLocationId: locationId }, () =>
-      this.refreshMapSelectedSamples()
+    this.setState({ mapPreviewedLocationId: locationId }, () =>
```

Comment:
Curious about the `selected` -> `preview` name change?

Reply:
I wrote this in the description but it's to clarify which samples are previewed (displayed on the right sidebar) and selected (checkmark checked on the right sidebar)